### PR TITLE
Check iowait

### PR DESCRIPTION
**Pull request by hanynowsky**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass 

#### New Plugins

- [ ] Tests

- [x] Add the plugin to the README

- [x] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.0.1] - 2016-11-28
```
Comment (majormoses) [inline CHANGELOG.md:8]: please remove the version we will bump when releasing.

File: bin/check-iowait.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+
+#
+# Check IO WAIT
+#
+class CheckIOWAIT < Sensu::Plugin::Check::CLI
+  option :warning, long: '--warning WARNING', short: '-w WARNING', description: 'warning percentage', proc: proc(&:to_f), default: 50
```
Comment (majormoses) [inline bin/check-iowait.rb:36]: can you break these onto multiple lines? It makes it easier to read

File: bin/check-iowait.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  # iw1 = `head -n1 /proc/stat|awk '{print $6}'`
+  def compute
+    ticker = `getconf CLK_TCK`
+    iv = config[:sleep] # number of sleep seconds
+    ut0 = File.read('/proc/uptime').split(' ')[0]
+    iw0 = File.foreach('/proc/stat').first.split(' ')[5]
+    sleep(iv)
+    ut1 = File.read('/proc/uptime').split(' ')[0]
+    iw1 = File.foreach('/proc/stat').first.split(' ')[5]
+    nbproc = if special_os
```
Comment (majormoses) [inline bin/check-iowait.rb:68]: Is there a reason you can't parse `/proc/cpuinfo` in both cases in ruby without needing to shell out?

Comment (hanynowsky) [reply bin/check-iowait.rb:68]: In some platforms and os configurations, using cpuinfo would give wrong results or not even possible.
This is more of a generic version.

File: bin/check-iowait.rb (hunk 2 above)
Comment (hanynowsky) [reply bin/check-iowait.rb:36]: I was fooled by travis each time complaining about wrong alignment.
I'd try again then. Thanks

File: CHANGELOG.md (hunk 1 above)
Comment (hanynowsky) [reply CHANGELOG.md:8]: ok

File: bin/check-iowait.rb (hunk 2 above)
Comment (majormoses) [reply bin/check-iowait.rb:36]: The alignment is very subtle it must be perfectly aligned with the first char (`:`) meaning that its gonna be some number of tabs and a final space.

**Comment by majormoses** [comment]
@hanynowsky are you planning on picking this back up?

File: bin/check-iowait.rb (hunk 2 above)
Comment (majormoses) [reply bin/check-iowait.rb:36]: Btw I updated this to reflect what is desired: https://github.com/sensu-plugins/documentation/blob/master/docs/developer_guidelines.md#coding-style

**Comment by majormoses** [comment]
closing due to inactivity, please reply back if you would like to see this through so I can re-open.

**Pull request closed without merging**